client/components/mma/upgrade/UpgradeSupportStyles.ts

```diff
@@ -0,0 +1,48 @@
+import { css } from '@emotion/react';
+import { brand, headline, space, textSans } from '@guardian/source-foundations';
+
+export const sectionSpacing = css`
+	margin-top: ${space[4]}px;
+`;
+
+export const linkCss = css`
+	${textSans.medium()};
+	color: ${brand[400]};
+	font-weight: 700;
+	text-decoration-line: underline;
+	margin-top: 10px;
+	margin-left: 20px;
```

Comment:
![image](https://github.com/guardian/manage-frontend/assets/114918544/6b52b84b-2365-45d3-b011-7995443a0e19)
this has margin that means it's off center on the mobile view

Reply:
Any suggestions how to get around that?